ci-operator/config/openshift/ironic-agent-image/openshift-ironic-agent-image-release-4.16.yaml

```diff
@@ -15,8 +15,7 @@ images:
   to: ironic-agent
 promotion:
   to:
-  - disabled: true
```

Comment:
I'm ok with the change, but looking at how we did back to move to 4.16 https://github.com/openshift/release/pull/46676
We don't update the `openshift-ironic-agent-image-release-4.<previous release>yaml` 
Is this change really necessary? Also I'm a bit puzzled with the disabled we had before so maybe is ok...

Reply:
There was a failing CI job without this IIRC. From what I understand promotion for only the latest release should be disabled, and it should be enabled for everything else. For the PR #46676, the promotion was already disabled for OCP 4.15 ([link](https://github.com/elfosardo/release/blob/1d81405463d77b9e40d3a5f2621382795380bf54/ci-operator/config/openshift/ironic-agent-image/openshift-ironic-agent-image-release-4.15.yaml#L17)).

Reply:
(I basically went to he origin branch of the PR, made sure the last commit hash is the same as the PR, and found the file for 4.15.)